benchmark/scripts/Benchmark_Driver

```diff
@@ -227,24 +236,28 @@ def submit(args):
     submit_to_lnt(data, args.lnt_host)
     return 0
 
+
 def run(args):
     optset = args.optimization
     file = os.path.join(args.tests, "Benchmark_" + optset)
     run_benchmarks(file, benchmarks=args.benchmarks,
-            num_samples=args.iterations, verbose=True,
-            log_directory=args.output_dir,
-            swift_repo=args.swift_repo)
+                   num_samples=args.iterations, verbose=True,
+                   log_directory=args.output_dir,
+                   swift_repo=args.swift_repo)
```

Comment:
That would be a better way to format, indeed.

> I think this indentation rule and the 80 columns rule can co-exist.

I can't agree.  In the ArgumentParser case below, following this formatting rule pushes the code well beyond 80 columns without a reasonable way to re-format.